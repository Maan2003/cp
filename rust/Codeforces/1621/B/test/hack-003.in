1
8
1 3 5
2 4 2
4 13 6
6 6 4
1 10 8
9 10 1
7 15 6
7 8 7
