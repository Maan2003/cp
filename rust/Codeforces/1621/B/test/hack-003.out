5
7
11
11
11
11
11
11
